tests/phpunit/integration/Core/Util/Plugin_StatusTest.php

```diff
@@ -11,58 +11,41 @@
 namespace Google\Site_Kit\Tests\Core\Util;
 
 use Google\Site_Kit\Core\Util\Plugin_Status;
+use Google\Site_Kit\Tests\FakeInstalledPlugins;
 use Google\Site_Kit\Tests\TestCase;
 
 /**
- * @group Assets
+ * @group Util
  */
 class Plugin_StatusTest extends TestCase {
-	/**
-	 * Initial active plugin state array.
-	 */
-	private $initial_active_plugins_state;
-
-	public function set_up() {
-		parent::set_up();
-
-		$this->initial_active_plugins_state = $GLOBALS['wp_tests_options']['active_plugins'];
```

Comment:
This global only affects the WP test env bootstrap, so it doesn't make sense to preserve/restore/modify.